Data/Migrations/20190307195457_init.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Since you didn't make any database changes, you shouldn't need this file or the Designer file